Dockerfile

```
@@ -442,6 +442,32 @@ ENV PATH="/usr/libexec/rust/bin:$PATH" LD_LIBRARY_PATH="/usr/libexec/rust/lib"
 
 # =^..^=   =^..^=   =^..^=   =^..^=   =^..^=   =^..^=   =^..^=   =^..^=   =^..^=
 
+FROM sdk-libc as sdk-bootconfig
+
+USER root
+
+ARG KERNELVER="5.10.109"
+
+RUN \
+  mkdir -p /usr/libexec/tools /usr/share/licenses/bootconfig && \
+  chown -R builder:builder /usr/libexec/tools /usr/share/licenses/bootconfig
+
+USER builder
+WORKDIR /home/builder
+COPY ./hashes/kernel /home/builder/hashes
+RUN \
+  sdk-fetch /home/builder/hashes && \
+  tar -xf linux-${KERNELVER}.tar.xz && rm linux-${KERNELVER}.tar.xz
+
+WORKDIR /home/builder/linux-${KERNELVER}
+RUN \
+  cp -p COPYING LICENSES/preferred/GPL-2.0 LICENSES/exceptions/Linux-syscall-note /usr/share/licenses/bootconfig
```

The Linux syscall exception license doesn't seem to apply to `bootconfig`.

```suggestion
  cp -p COPYING LICENSES/preferred/GPL-2.0 /usr/share/licenses/bootconfig
```